etc/kubernetes/helm/enterprise-gateway/Chart.yaml

```diff
@@ -1,10 +1,11 @@
+apiVersion: v2
 name: enterprise-gateway
 description: A helm chart to deploy Jupyter Enterprise Gateway
-version: 3.0.0.dev0
-apiVersion: v1
+version: 4.0.0
```

Comment:
This should probably be  reverted to `3.0.0.dev0` since the [release script will update this](https://github.com/jupyter-server/enterprise_gateway/blob/67683767656af1e41879c609159edfedcab3cfe3/release.sh#L206).  Are you finding this particular version format to be problematic when working from `main`?

Reply:
linter doesn't like it:
```
helm lint etc/kubernetes/helm/enterprise-gateway
==> Linting etc/kubernetes/helm/enterprise-gateway
[ERROR] Chart.yaml: version '3.0.0.dev0' is not a valid SemVer
[ERROR] templates/: validation: chart.metadata.version "3.0.0.dev0" is invalid
[ERROR] : unable to load chart
	validation: chart.metadata.version "3.0.0.dev0" is invalid

Error: 1 chart(s) linted, 1 chart(s) failed
```

Reply:
I see, and you're working from the primary branch.

Let's go ahead and change this to be the "next" release (i.e., drop the ".devx" suffix).  This will require a change to the release script when setting the dev version back.

I'll open an issue for this, but we can go ahead and set it to `3.0.0` since that is the equivalent change.
(I see you already reverted it to the dev version - no worries as long as you're okay manually updating for now.)

Reply:
i also added this:

Reply:
Do you happen to know if `appVersion` is as strict as `version`?  i.e., could we use `version` as the actual chart version and let `appVersion` reflect the release?  If it isn't as strict, then we could retain the `.dev` suffix.

Reply:
From the docs: `appVersion: The version of the app that this contains (optional). Needn't be SemVer. Quotes recommended.`

So we can have `appVersion` represent the EG release and contain values like `"3.0.0.dev0"` and we'll let `version` be static relative to the file.  I think it would be fine to start that value as `3.0.0`.

I will handle the appropriate "release processing" via the other issue (that will be linked to this PR).

Reply:
that is correct